core_chat_presentation/src/commonMain/kotlin/net/thechance/mena/core_chat/presentation/screens/contacts/ContactsScreen.kt

```diff
@@ -0,0 +1,185 @@
+                        ).clickable(
+                            interactionSource = remember { MutableInteractionSource() },
+                            indication = null
+                        ) {
+                            onResyncClick()
+                        },
+                    contentAlignment = Alignment.Center
+                ) {
+                    Icon(
+                        painter = painterResource(Res.drawable.resync),
+                        contentDescription = null,
+                        modifier = Modifier.size(20.dp),
+                        tint = Theme.colorScheme.shadePrimary,
+                    )
+                }
+            }
+        )
+        LazyColumn(
+            modifier = Modifier.fillMaxWidth().padding(horizontal = Theme.spacing._16),
+            verticalArrangement = Arrangement.spacedBy(Theme.spacing._16),
+            contentPadding = PaddingValues(vertical = Theme.spacing._8)
+        ) {
+            items(20) {
+                ContactItem(
+                    contact = Contact(
+                        id = "",
+                        firstName = "Mona",
+                        lastName = "Ayman Elsayed Ahhmed",
+                        phoneNumbers = listOf("01224479626"),
+                        imageUri = null,
+                    ),
+                    hasAccount = true,
+                    hasImage = true,
+                    onContactClick = {},
+                )
+            }
+        }
+    }
+}
+
+@Composable
+private fun ContactItem(
+    contact: Contact,
+    hasAccount: Boolean,
+    hasImage: Boolean,
+    onContactClick: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    Row(
+        modifier = modifier.fillMaxWidth()
+            .clickable(
+                interactionSource = remember { MutableInteractionSource() },
+                indication = null
+            ) { onContactClick() },
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        Avatar(
+            imageUri = contact.imageUri,
+            initials = contact.initials,
+            size = 48
+        )
+        Column(
+            modifier = Modifier.padding(start = Theme.spacing._8).weight(1f)
+        ) {
+            Text(
+                text = contact.displayName,
+                style = Theme.typography.label.large,
+                color = Theme.colorScheme.shadePrimary,
+                maxLines = 1,
+                overflow = TextOverflow.Ellipsis
+            )
+            Text(
+                text = contact.phoneNumbers.firstOrNull() ?: "",
+                style = Theme.typography.label.medium,
+                color = Theme.colorScheme.shadeTertiary,
+                maxLines = 1,
+                overflow = TextOverflow.Ellipsis,
+                modifier = Modifier.padding(top = Theme.spacing._2)
+            )
+
+        }
+        if (hasAccount) {
+            if (hasImage) {
```

Comment:
what if we change this to be like this ?

```suggestion
        if (hasAccount) {
            val iconRes = if (hasImage) Res.drawable.image_enabled else Res.drawable.image_disabled
            Icon(
                painter = painterResource(iconRes),
                contentDescription = null,
                modifier = Modifier.padding(start = Theme.spacing._8).size(24.dp),
                tint = Color.Unspecified
            )
        }
    }
```